ios/Openid4vpBle/Openid4vpBle.swift

```diff
@@ -3,20 +3,20 @@ import Foundation
 @available(iOS 13.0, *)
 @objc(Openid4vpBle)
 class Openid4vpBle: RCTEventEmitter {
-    
+    var tuvaliVersion: String = " "
```

Comment:
Should be consistent with Android default value set

Reply:
changed to unknown.